iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/DanglingLineData.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.util;
+
+import org.apache.commons.math3.complex.Complex;
+import com.powsybl.iidm.network.DanglingLine;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public class DanglingLineData {
+
+    private final String id;
+
+    private final double r;
+    private final double x;
+    private final double g1;
+    private final double g2;
+    private final double b1;
+    private final double b2;
+
+    private final double u1;
+    private final double theta1;
+    private final double p0;
+    private final double q0;
+
+    double boundaryBusU;
+    double boundaryBusTheta;
+
+    public DanglingLineData(DanglingLine danglingLine) {
+
+        id = danglingLine.getId();
+        r = danglingLine.getR();
+        x = danglingLine.getX();
+        g1 = danglingLine.getG() / 2.0;
+        b1 = danglingLine.getB() / 2.0;
+        g2 = danglingLine.getG() / 2.0;
+        b2 = danglingLine.getB() / 2.0;
+        p0 = danglingLine.getP0();
+        q0 = danglingLine.getQ0();
+
+        u1 = getV(danglingLine);
+        theta1 = getTheta(danglingLine);
+
+        boundaryBusU = Double.NaN;
+        boundaryBusTheta = Double.NaN;
+
+        Complex v1 = new Complex(u1 * Math.cos(theta1), u1 * Math.sin(theta1));
```

Comment:
We could use 
```
angle=getAngle(danglingLine)
Complex v1 = ComplexUtils.polar2Complex(v, Math.toRadians(angle));
```
So the conversion degrees -> radians is in a single point, and the creation of complex from polar coordinates is explicit.

Reply:
Solved